helm/happa/values.yaml

```diff
@@ -43,6 +43,8 @@ gcp:
 
 ingress:
   tls:
+    letsencrypt: false
```

Comment:
its a little bit weird that the default is `false` since `dex-app` and `athena` has default `true`, but keeping same value not to mess with the state